src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -4362,6 +4362,12 @@ public class B
                 // (7,27): error CS1016: Named attribute argument expected
                 //     [A(3, z: 5, X = 6, y: 1)]
                 Diagnostic(ErrorCode.ERR_NamedArgumentExpected, "1").WithLocation(7, 27),
+                // (8,17): error CS1738: Named argument specifications must appear after all fixed arguments have been specified. Please use language version 7.2 or greater to allow non-trailing named arguments.
+                //     [A(3, z: 5, 1)]
+                Diagnostic(ErrorCode.ERR_NamedArgumentSpecificationBeforeFixedArgument, "1").WithArguments("7.2").WithLocation(8, 17),
```

Comment:
>Diagnostic(ErrorCode.ERR_NamedArgumentSpecificationBeforeFixedArgument, "1").WithArguments("7.2").WithLocation(8, 17) [](start = 16, length = 117)

If we don't want this test to break once we bump the ```default``` version, we should use explicit 7.1 version for this test. Consider making this change for this test and other existing tests that got affected by this error.